packages/serverpod/lib/src/server/server.dart

```diff
@@ -254,7 +254,7 @@ class Server {
     } else if (uri.path == '/v1/websocket') {
       await _dispatchWebSocketUpgradeRequest(
         request,
-        MethodWebsocketRequestHandler().handleWebsocket,
+        MethodWebsocketRequestHandler.handleWebsocket,
```

Comment:
Question: Since you implemented it as a local variable, does it even have to be static? Seems like we could just create one and use the local variable like we do now?

Reply:
To keep it consistent with how `EndpointWebsocketRequestHandler` works above I changed it like this 👍